.azure-pipelines/baseline_test/baseline.test.template.yml

```diff
@@ -8,79 +8,77 @@ jobs:
           TESTBED_PREP_TOPOLOGY: t0
           CHECKER: t0_checker
           TOPOLOGY: t0
           PREPARE_TIME: 30
-          NUM_ASIC: 1
-          DEPLOY_MG_EXTRA_PARAMS: ""
-          COMMON_EXTRA_PARAMS: ""
-        t0_2vlans:
+        impacted-area-kvmtest-t0-2vlans:
           TESTBED_PREP_TOPOLOGY: t0-2vlans
           CHECKER: t0-2vlans_checker
           DEPLOY_MG_EXTRA_PARAMS: "-e vlan_config=two_vlan_a "
           TOPOLOGY: t0
           PREPARE_TIME: 30
-          NUM_ASIC: 1
-          COMMON_EXTRA_PARAMS: ""
-        t1_lag:
+        impacted-area-kvmtest-t1-lag:
           TESTBED_PREP_TOPOLOGY: t1
           CHECKER: t1_checker
           TOPOLOGY: t1-lag
           # 50 mins for preparing testbed, 30 mins for pre-test and post-test
           PREPARE_TIME: 80
-          NUM_ASIC: 1
-          COMMON_EXTRA_PARAMS: ""
-          DEPLOY_MG_EXTRA_PARAMS: ""
-        dualtor:
+        impacted-area-kvmtest-dualtor:
           TESTBED_PREP_TOPOLOGY: dualtor
           CHECKER: dualtor_checker
           COMMON_EXTRA_PARAMS: "--disable_loganalyzer "
           TOPOLOGY: dualtor
           # 30 mins for preparing testbed, 30 mins for pre-test and 20 mins for post-test
           PREPARE_TIME: 80
-          NUM_ASIC: 1
-          DEPLOY_MG_EXTRA_PARAMS: ""
-        t1_multi_asic:
-          TESTBED_PREP_TOPOLOGY: t1-multi-asic
+        impacted-area-kvmtest-multi-asic-t1:
+          TESTBED_PREP_TOPOLOGY: multi-asic-t1
           CHECKER: t1-multi-asic_checker
           TOPOLOGY: t1-8-lag
           NUM_ASIC: 4
           PREPARE_TIME: 30
-        sonic_t0:
+        impacted-area-kvmtest-t0-sonic:
           TESTBED_PREP_TOPOLOGY: t0-sonic
           CHECKER: t0-sonic_checker
           COMMON_EXTRA_PARAMS: "--neighbor_type=sonic "
           TOPOLOGY: t0-64-32
           PREPARE_TIME: 40
           VM_TYPE: vsonic
-          NUM_ASIC: 1
-          DEPLOY_MG_EXTRA_PARAMS: ""
-        dpu:
+          SPECIFIC_PARAM: '[
+            {"name": "bgp/test_bgp_fact.py", "param": "--neighbor_type=sonic --enable_macsec --macsec_profile=128_SCI,256_XPN_SCI"},
+            {"name": "macsec", "param": "--neighbor_type=sonic --enable_macsec --macsec_profile=128_SCI,256_XPN_SCI"}
+            ]'
+        impacted-area-kvmtest-dpu:
           TESTBED_PREP_TOPOLOGY: dpu
           CHECKER: dpu_checker
           TOPOLOGY: dpu
           PREPARE_TIME: 30
-          NUM_ASIC: 1
-          COMMON_EXTRA_PARAMS: ""
-          DEPLOY_MG_EXTRA_PARAMS: ""
+          SPECIFIC_PARAM: '[
+            {"name": "dash/test_dash_vnet.py", "param": "--skip_dataplane_checking"}
+            ]'
     condition: contains(dependencies.get_impacted_area.outputs['SetVariableTask.PR_CHECKERS'], variables['CHECKER'])
     variables:
       TEST_SCRIPTS: $[ dependencies.get_impacted_area.outputs['SetVariableTask.TEST_SCRIPTS'] ]
+      NUM_ASIC: 1
+      VM_TYPE: ceos
+      COMMON_EXTRA_PARAMS: ""
+      DEPLOY_MG_EXTRA_PARAMS: ""
+      SPECIFIC_PARAM: "[]"
```

Comment:
These are template variables, not job variables. So I don't think here is the right place to set these variables.

Reply:
I set here as default value, if any job has defined specific value, it can be covered, I tested the variables is working, you can refer to test plan info in trigger test https://dev.azure.com/mssonic/build/_build/results?buildId=815438&view=logs&j=aa7053a2-02f1-5ee6-62fc-702e8a17afaa&t=89f83fc5-5503-5ce3-3c96-90268536c3da, all specific values are passed correctly

Reply:
You can set the default value in calculate-instance-numbers.yml

Reply:
These params are used for `run-test-elastictest-template.yml`, `calculate-instance-numbers.yml` won't use these params, so from the code logic, I don't think we should be set in `calculate-instance-numbers.yml`

Reply:
Can you set them in run-test-elastictest-template.yml, I think which is better.

Reply:
We need to set a default value, or we will get error with param import `VM_TYPE: $(VM_TYPE)`, that's the weak point of job matrix